website/versioned_docs/version-5.x/ecosystem-angular.md

```diff
@@ -773,6 +773,17 @@ If you're looking for a quick one-liner, try adding this line near the top of yo
 <script src="https://unpkg.com/core-js-bundle/minified.js"></script>
 ```
 
+To correct the error `It looks like your application or one of its dependencies is using i18n`.
+
+Install `@angular/localize` in your root-config module
+```sh
+npm i @angular/localize
+```
+Add following import to your root-config.js
+```ts
+import "@angular/localize/init";
```

Comment:
Hmm it might be best to add a note that `@angular/localize` (and other angular libs) should be shared dependencies, to avoid redownloading them in root config and in other MFEs. However, that can be done in a follow-up PR. Please send that follow up PR if interested in contributing further.